Resources/Private/JavaScript/store.svelte.ts

```
@@ -1,23 +1,71 @@
-export const wizardConfigStore = writable(null);
+export type ShapeType = "rect" | "polygon";
 
-export const focuspoints = writable([]);
+type Focuspoint = {[K in string]: string} & {
+  __shape: ShapeType;
+  __data: any;
+}
+
+type ShapeConfig = {
+  component: Function;
+  constructor(config: WizardConfig): object;
```

Biome (1.9.4)

[error] Don't use the constructor method in type aliases.

Type aliases simply rename types. They don't execute code, so a constructoris misleading.

(lint/suspicious/noMisleadingInstantiator)